### DISCUSSION TITLE
Add Funding section with OpenCollective and Drips.io via OC

### DISCUSSION THREAD

**Discussion by tony-stark-eth** [mikro-orm/mikro-orm#6075] [Ideas]
**Is your feature request related to a problem? Please describe.**
Github Sponsor is somewhat limited and only reaches a certain Group. 

**Describe the solution you'd like**
Similar to Svelte create a Funding section in Readme.md and refer to OpenCollective, and configure Drips.io if you into that. 

**Additional context**
Love what you're building, especially because I come from a PHP Background with Doctrine, and I'm so glad I found this ORM which has similar Patterns like Unit of Work, so I would like to see that you get every funding you can get to compete with lesser ORMs like Drizzle.

**Comment by tony-stark-eth** [comment]
Besides that I created an Adapter for Lucia Auth which hopefully brings more Visibility :)

**Comment by tony-stark-eth** [comment]
You can find Sveltes example here https://github.com/sveltejs/kit?tab=readme-ov-file#supporting-svelte

**Comment by B4nan** [comment]
I am not sure if adding more options would make a huge difference, what I am sure of is that it's quite a lot of weird paperwork to set it up, pretty much all those services are US based, you need to go through things like some US specific forms which I had no idea how to fill in. Even having an org account on GH sponsors was a blocker for me, since they ask for things I simply have no idea how to translate to EU (or better say czech) terms.

Maybe things changed, it's been some time since I tried last time, but I am personally more than happy with GH sponsors and all the other solutions would most likely be harder for me (both to setup and to consume). Do you personally have any experience with them (as a maintainer/owner, not a user/contributor)? 

Those conversations are always a bit funny, I recall many people asking me to add ways to sponsor, yet once it was there, they went silent and never contributed 🙃 Why do you consider GH sponsors to be problematic?

Agreed that we should have some section in the docs and in the readme.

**Reply by tony-stark-eth** [reply]
Sadly no experience as a maintainer/owner, probably that is the reason of my naive approach. 😅 

GH sponsor is problematic for me personally because you can contribute starting with 10$, and only in $, which a credit card would be mandatory for, and I may have one without a conversion fee but its still a hassle.

So when I want to contribute with another payment option or € I cannot do this, and I think 10$ minimum is too high, not sure if thats a GH Sponsor thing.

Docs and README.md would be an improvement though because that I can see on mobile and is not easily unnoticed as the small sidebar panel for GH Sponsors.

**Reply by B4nan** [reply]
> and I think 10$ minimum is too high, not sure if thats a GH Sponsor thing.

I see there is some hard limit of 10$, maybe its their thing, maybe its based on the lowest tier. I did set up the lowest tier to 10$, I had the 5$ too back in the day, but its really too low to be even visible for me.  And nope, it didn't work in the way that I would have lots of such sponsors, I don't think I ever had more than 10 people on this lowest tier. 

Note that you can always go with a one-time sponsorship, which is still limited to 10$, but again, if 10$ one time is too much, maybe try contributing another way than with money, in the end, that helps even more than a few bucks.

> and only in $

The price is in dollars, but it's not relevant what currency you have on your own account. Do you think I have a $ account? Nope, I don't, I am receiving the funds in euros (just like I can pay anywhere regardless of what currency they charge). Not sure what you mean by the need for credit card either, you can pay with a debit card too, and I don't expect any difference with other providers here. Or you would rather use a wire transfer?
